doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -290,6 +290,13 @@ nRF Desktop
     The PDK can act as a sample mouse or keyboard.
     It supports the Bluetooth LE HID data transport and uses SoftDevice Link Layer with Low Latency Packet Mode (LLPM) enabled.
     The PDK uses MCUboot bootloader built in the direct-xip mode (``MCUBOOT+XIP``) and supports firmware updates using the :ref:`nrf_desktop_dfu`.
+  * Support for the nRF54H20 DK with the ``nrf54h20dk/nrf54h20/cpuapp`` board target.
+    The PDK can act as a sample mouse.
+    It supports the Bluetooth LE HID data transport and uses SoftDevice Link Layer with Low Latency Packet Mode (LLPM) enabled.
+    Application uses new USB stack "USB-next".
+    It allows using USB High-Speed with report rate up to 8000 Hz.
+    The nRF54H20 DK uses DFU and secure boot method called the Software Updates for Internet of Things (SUIT).
```

Comment:
`The nRF54H20 DK supports firmware update using Software Updates for Internet of Things (SUIT).`
Not sure if we should call it `secure boot` as we do not sign our update images now.